content/pt-br/docs/reference/kubectl/kubectl.md

```diff
@@ -0,0 +1,438 @@
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Se estiver presente, o scope do namespace para essa requisição do CLI. 
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--one-output</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"> Se true, escreve logs apenas para seus níveis de gravidade (enquanto também escreve para cada nível de gravidade menor)</td>
+</tr>
+
+<tr>
+<td colspan="2">--password string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+Senha para autenticação básica ao servidor da API.
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--profile string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "none"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+Nome do perfil a ser capturado. Um dentre (none|cpu|heap|goroutine|threadcreate|block|mutex).
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--profile-output string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "profile.pprof"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+Nome do arquivo no qual será escrito o perfil.
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--request-timeout string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "0"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+O tempo a ser esperado antes de desistir de uma única requisição do servidor. Valores diferente de 0 devem conter unidade de tempo correspondente (e.g. 1s, 2m, 3h). O valor 0 significa não dar timeout nas requisições.
+.</td>
+</tr>
+
+<tr>
+<td colspan="2">-s, --server string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">O endereço e porta do servidor API Kubernetes</td>
+</tr>
+
+<tr>
+<td colspan="2">--skip-headers</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Se true, evita prefixos de cabeçalho nas mensagens de log.
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--skip-log-headers</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Se true, evita cabeçalhos quando abrir arquivos de log. 
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--stderrthreshold severity&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 2</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+Logs acima ou nesse limiar irão para o stderr
+</td>
+</tr>
+
+<tr>
+<td colspan="2">--tls-server-name string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Nome do servidor para usar a validação do certificado do servidor. Se não for fornecido, o nome do host usado para contatar o servidor é usado</td>
+</tr>
+
+<tr>
+<td colspan="2">--token string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Bearer token para autenticação para o servidor API</td>
+</tr>
+
+<tr>
+<td colspan="2">--user string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">O nome do usuário do kubeconfig a usar</td>
```

Comment:
I think we can use something like: `O nome de usuário do kubeconfig a ser usado`